### PR TITLE
Add wdio tests for terra-clinical-onset-picker

### LINKED ISSUE #303: Add webdriver.io tests and remove nightwatch tests
# Issue Description
... for the following packages.

- [x] `terra-clinical-item-view` (https://github.com/cerner/terra-clinical/pull/304)
- [x] `terra-clinical-label-value-view` (https://github.com/cerner/terra-clinical/pull/307)
- [x] `terra-clinical-onset-picker` (https://github.com/cerner/terra-clinical/pull/308)
- [x] `terra-clinical-no-data-view` (https://github.com/cerner/terra-clinical/pull/309)
- [x] `terra-clinical-item-display` (Remove nightwatch tests only) (https://github.com/cerner/terra-clinical/pull/309)
- [x] `terra-clinical-error-view` (Remove nightwatch tests only) (https://github.com/cerner/terra-clinical/pull/309)

## Issue Type

- [ ] New Feature
- [X] Enhancement
- [ ] Bug
- [ ] Other

### PR DESCRIPTION
**Pull request by rakshith91**
### Summary
... and remove nightwatch tests.

Resolves #303 
Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by ryanthemanuel** [comment]
We need to fix the build before we proceed.

File: packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ describe('Onset Picker', () => {
       browser.clearElement('[data-terra-clinical-onset-picker="age"] input');
       browser.setValue('[data-terra-clinical-onset-picker="age"] input', 10);
       browser.click('button[type="submit"]');
+      browser.pause(5000);
```
Comment (emilyrohrbough) [inline packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js]: Why are we adding this pause?

Comment (rakshith91) [reply packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js]: The snapshots generated are highly inconsistent without the pause. The alert takes a second to load completely due to the animation.

Comment (ryanthemanuel) [reply packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js]: I thought there were some webdriverIO commands that would automatically wait until the element was on screen.  Can we use one of those instead before doing the click?  (http://webdriver.io/api/utility/waitForVisible.html or something similar)

Comment (rakshith91) [reply packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js]: I tried that earlier. `waitForVisible` ( and other similar commands) do not recognize the completion of the animation.
Here's an example of the diff when pause is not used.

image

**Comment by emilyrohrbough** [comment]
Screenshots are cutting off the popup in:
- https://github.com/cerner/terra-clinical/blob/92421e1c5997090ef6a5872ffebaecf71ed1dc46/packages/terra-clinical-onset-picker/tests/wdio/__snapshots__/reference/chrome/Cannot_select_more_than_24_months%5Bdefault%5D.622x768.png
- https://github.com/cerner/terra-clinical/blob/92421e1c5997090ef6a5872ffebaecf71ed1dc46/packages/terra-clinical-onset-picker/tests/wdio/__snapshots__/reference/chrome/Cannot_select_number_of_months_that_exceede_age%5Bdefault%5D.622x768.png
- https://github.com/cerner/terra-clinical/blob/92421e1c5997090ef6a5872ffebaecf71ed1dc46/packages/terra-clinical-onset-picker/tests/wdio/__snapshots__/reference/chrome/Cannot_select_number_of_weeks_that_exceede_age%5Bdefault%5D.622x768.png
- https://github.com/cerner/terra-clinical/blob/92421e1c5997090ef6a5872ffebaecf71ed1dc46/packages/terra-clinical-onset-picker/tests/wdio/__snapshots__/reference/chrome/Cannot_select_number_of_years_greater_than_age%5Bdefault%5D.622x768.png
- https://github.com/cerner/terra-clinical/blob/92421e1c5997090ef6a5872ffebaecf71ed1dc46/packages/terra-clinical-onset-picker/tests/wdio/__snapshots__/reference/chrome/Cannot_select_number_of_weeks_that_exceede_age%5Bdefault%5D.622x768.png

**Pull request merged**